src/nfa/operations.cc

```diff
@@ -179,10 +190,13 @@ bool Mata::Nfa::are_state_disjoint(const Nfa& lhs, const Nfa& rhs)
     return true;
 } // are_disjoint }}}
 
+//Should be in-place method? I would still name it complete, it is a verb too.
+//Alphabet should be optional, the automaton has an alphabet that would be used implicitly.
 bool Mata::Nfa::make_complete(Nfa& aut, const Alphabet& alphabet, State sink_state) {
```

Comment:
The same comment are already by the function declaration. Let us resolve this suggestion there.

Reply:
ok